xml/installation-installation-gui_installer.xml

```diff
@@ -410,32 +786,21 @@
    </para>
   </listitem>
  </itemizedlist>
- <note>
-  <para>
-   Because the GUI installer is disabled after deployment is complete, whether
-   you installed your cloud via the GUI or via the command line, if you want to
-   re-enable the GUI later but are not re-installing from scratch, run these
-   commands:
-  </para>
-<screen>sudo a2ensite dayzero-apache.conf
-sudo systemctl start dayzero.service</screen>
- </note>
- </section>
+  </section>
 <section>
-<title>Final Steps</title>
- <itemizedlist>
-  <listitem>
-   <para>
-    The public Horizon Dashboard connection is over TLS, thus please use
-    HTTPS:// to navigate to Horizon with the IP address the installer presents
-    once install is complete.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    Visit the <xref linkend="ceph_overview"/> for Ceph post-install steps.
-   </para>
-  </listitem>
- </itemizedlist>
+<title>Use the Install UI securely</title>
+ <para>
+  The Install UI is a web-based application that runs on the &lcm; node and is
+  implemented in ReactJS and Python. This application runs behind Apache 2.0 on
+  port 3000. Browser refresh is not necessary if you are using the browser
+  connected directly to the Install UI.
```

Comment:
~Nobody cares about these implementation details -- ReactJS, Python, and Apache don't need to be mentioned.  (The people that do care will just take a look at the source code.)

I also don't understand the bit about refreshing the browser/page(?).

Reply:
Because of the network isolation of the deployer from the rest of a customers cloud, they may access the Installation UI 2 ways:
1. by logging into the lifecycle manager host directly and launching a browser from there
2. by creating an ssh tunnel from their local server to the lifecycle manager host

during the deployment process the network on the lifecycle manager will be restarted. If the customer is connected via method 2 above , then they will lose that connection when the network restarts, and will need to refresh/reload their browser after re-establishing their ssh tunnel. If they launched the browser directly from the lifecycle manager, then they will not have to manually refresh .